llvm/lib/Target/RISCV/RISCVInstrInfo.cpp

```diff
@@ -821,40 +821,21 @@ bool RISCVInstrInfo::isAsCheapAsAMove(const MachineInstr &MI) const {
 }
 
 Optional<DestSourcePair>
-RISCVInstrInfo::getCMovReg(MachineInstr &MI) const{
-  switch(MI.getOpcode()) {
```

Comment:
The change here looks like it will conflict with https://github.com/plctlab/llvm-project/pull/25 when merging